src/object/obj_internal.h

```diff
@@ -52,6 +52,8 @@
 extern bool	cli_bypass_rpc;
 /** Switch of server-side IO dispatch */
 extern bool	srv_io_dispatch;
+extern bool srv_enable_dtx;
+extern uint64_t srv_start_time;
```

Comment:
the usage of srv_start_time seems unreasonable based on previous discussion about HLC, maybe can remove it for now.

Reply:
It can be removed as explained below.